apkid/rules/dex/obfuscators.yara

```diff
@@ -187,3 +187,23 @@ rule allatori_demo : obfuscator
   condition:
     $s and is_dex
 }
+
+rule aamo : obfuscator
+{
+  meta:
+    description = "AAMO"
+    author = "P0r0"
+    url = "https://github.com/necst/aamo"
+    backup_url = "https://github.com/P0r0/aamo"
+    example1 = "c1ef860af0e168f924663630ed3b61920b474d0c8b10e2bde6bfd3769dbd31a8"
+    example2 = "eb0d4e1ba2e880749594eb8739e65aa21b6f7b43798f04b6681065b396c15a78"
+    example3 = "b1e20bf3bdc53972424560e20c6d9ad12e5e47b8ed429a77f4ba5ff6cb92cb27"
+    example4 = "82a570c272579aacdc22410e152f4519738f4e0ececa84e016201c33ad871fa6"
+
+  strings:
+    $a = { 00 0f 63 6f 6e 76 65 72 74 54 6f 53 74 72 69 6e 67 00 } // convertToString
```

Comment:
This seems very likely to FP, even with the null padding. On github, for java, there are 200k+ results for this term. It doesn't seem likely that this is from the ammo code.

Reply:
Yes but the definition of the rule is ```$a and $b and is_dex```. I mean, if one only checks for ```$a``` I understand that we will have a lot of FPs. But it is not the case right? May be I didn't properly understand how Yara rules work?